src/svelte/svelte.component.js

```diff
@@ -0,0 +1,256 @@
+import { showFrameworkObservable, getBorder } from 'common/colored-border.js';
```

Comment:
Why? Since it's part of the build step, won't it work fine after you run `npm run build`?